src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,10 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return sprintf(
+            'response (with status code %s) %s',
+            $other->getStatusCode(),
+            $this->toString()
```

Comment:
This is to improve the failure messages: `Failed asserting that response (with status code 400) is a hit`.